core/src/main/java/com/novoda/noplayer/internal/exoplayer/SkippedAdverts.java

```diff
@@ -32,6 +32,21 @@ static AdPlaybackState markAllNonPlayedAdvertsAsSkipped(List<AdvertBreak> advert
         return adPlaybackStateWithSkippedAdGroups;
     }
 
+    /**
+     * Transforms all adverts in the given break that are not currently Played to Skipped.
+     *
+     * @param advertBreaks    The client representation of all the adverts break.
+     * @param advertBreak    The client representation of the advert break to be skipped.
+     * @param adPlaybackState The {@link AdPlaybackState} to alter advert state on.
+     * @return The {@link AdPlaybackState} with the new Skipped states.
+     */
+    static AdPlaybackState markAllNonPlayedAdvertsAsSkipped(AdvertBreak advertBreak, List<AdvertBreak> advertBreaks, AdPlaybackState adPlaybackState) {
+        AdPlaybackState adPlaybackStateWithSkippedAdGroups = adPlaybackState;
+        int breakIndex = advertBreaks.indexOf(advertBreak);
+        adPlaybackStateWithSkippedAdGroups = adPlaybackStateWithSkippedAdGroups.withSkippedAdGroup(breakIndex);
+        return adPlaybackStateWithSkippedAdGroups;
```

Comment:
seems like we could just return the result on the previous line and we wouldn't need the extra `adPlaybackStateWithSkippedAdGroups` variable